### ISSUE TITLE
[mail] Failed installing 'openhab-action-mail'

### ISSUE THREAD

**Issue by JohnBeton22** [openhab/openhab2-addons#5792]
After updating to latest snapshot openHAB 2.5.0 Build #1628, I get this error in the logs after starting the LSP:

```
[INFO ] [thome.model.lsp.internal.ModelServer] - Started Language Server Protocol (LSP) service on port 5007
[ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-action-mail'
```

The Mail action is not visible anymore in Paper UI, Addons, Actions

**Comment by openhab-5iver** [comment]
https://github.com/openhab/openhab1-addons/pull/5852

You'll need to install the new Mail binding to use a mail Action.

**Comment by kaikreuzer** [comment]
Or activate the legacy addon support in the Paper UI or the addons.cfg.

**Comment by JohnBeton22** [comment]
Ok thanks!

**Comment by SiggiFR** [comment]
Hi, my way to get a clean system regarding mail action after upgrading to #1639:
1. installed new mail binding
2. PaperUI > Configuration > System > AddOn Management > Include Legacy 1.x Bindings set to ON
3. PaperUI > Add-Ons > Actions > Send-Mail action now visible as "installed". I uninstalled it.
4. Reverted Legacy binding from step 2. to OFF (optional task depending on other system requirements)
5. Updated all rules to new sendmail syntax ((https://www.openhab.org/addons/bindings/mail/))
Hope it helps someone.
